index.html

```diff
@@ -70,6 +70,163 @@ <h1>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+        <div class="project-card">
+          <div class="project-card_image">
+            
+          </div>
+          <div class="project-card_details">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="stacks">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+        <div class="project-card">
+          <div class="project-card_image">
+            
+          </div>
+          <div class="project-card_details">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="stacks">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+        <div class="project-card">
+          <div class="project-card_image">
+            
+          </div>
+          <div class="project-card_details">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="stacks">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+        <div class="project-card">
+          <div class="project-card_image">
+            
+          </div>
+          <div class="project-card_details">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="stacks">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+        <div class="project-card">
+          <div class="project-card_image">
+            
+          </div>
+          <div class="project-card_details">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="stacks">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <div class="about-me">
```

Comment:
- [ ] Following the best practices in using the correct semantic tags, the about me section should be inside a `section tag` and not a `div`, I'll suggest you update this :+1: :waffle:

Reply:
@V-Blaze sorted :100: :+1: :smile: